src/main/java/com/rtb/projectmanagementtool/user/AuthServlet.java

```diff
@@ -0,0 +1,91 @@
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import com.google.sps.data.Log;
+import java.io.IOException;
+import java.util.*;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/user")
+public class UserServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+
+    List<Log> Logs = new ArrayList<>();
+    String status; // logged in or out
+    String userEmail;
+    long AuthID; // ID to track user for storing their info in datastore
+    String urlToRedirectToAfterUserLogsIn; // go to index (all user data should load)
+    String urlToRedirectToAfterUserLogsOut; // go to index with other pages blocked
+    String loginUrl;
+    String logoutUrl;
+
+    // get all the properties
+    if (userService.isUserLoggedIn()) {
+      status = "In";
+      userEmail = userService.getCurrentUser().getEmail();
+      AuthID = userService.getCurrentUser().getId();
```

Comment:
Apparently getId() isn't a valid method (got it from a tutorial but I think it was for something else now that I look at it again). If anyone has a tutorial or suggestions lmk  :)

Reply:
`getUserId` https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/users/User.html#getUserId--


I'm curious though how exactly we're planning on using this and how it connects to all the other servlets.  This might be worth explaining in the design doc before implementing

Reply:
Wrote more info on Design Doc.

Reply:
Lmk what you all think